app/models/membership.rb

```diff
@@ -84,7 +84,7 @@ def as_json(options={})
   end
 
   def get_valid_resource
```

Comment:
Same as before, expert can't manage the membership and return the membership is a error